packages/trader/src/AppV2/Utils/trade-params-utils.tsx

```diff
@@ -8,25 +8,84 @@ import {
 } from '@deriv/shared';
 
 export const getTradeParams = (symbol?: string) => ({
-    [TRADE_TYPES.RISE_FALL]: ['duration', 'stake', 'allow_equals'],
-    [TRADE_TYPES.RISE_FALL_EQUAL]: ['duration', 'stake', 'allow_equals'],
-    [TRADE_TYPES.HIGH_LOW]: ['trade_type_tabs', 'duration', 'barrier', 'stake'],
-    [TRADE_TYPES.TOUCH]: ['trade_type_tabs', 'duration', 'barrier', 'stake'],
-    [TRADE_TYPES.MATCH_DIFF]: ['last_digit', 'duration', 'stake'],
-    [TRADE_TYPES.EVEN_ODD]: ['duration', 'stake'],
-    [TRADE_TYPES.OVER_UNDER]: ['last_digit', 'duration', 'stake'],
-    [TRADE_TYPES.ACCUMULATOR]: ['growth_rate', 'stake', 'take_profit', 'accu_info_display'],
-    [TRADE_TYPES.MULTIPLIER]: [
-        'multiplier',
-        'stake',
-        'risk_management',
-        ...(shouldShowExpiration(symbol) ? ['expiration'] : []),
-        'mult_info_display',
-    ],
-    [TRADE_TYPES.TURBOS.LONG]: ['trade_type_tabs', 'duration', 'payout_per_point', 'stake', 'take_profit'],
-    [TRADE_TYPES.TURBOS.SHORT]: ['trade_type_tabs', 'duration', 'payout_per_point', 'stake', 'take_profit'],
-    [TRADE_TYPES.VANILLA.CALL]: ['trade_type_tabs', 'duration', 'strike', 'stake'],
-    [TRADE_TYPES.VANILLA.PUT]: ['trade_type_tabs', 'duration', 'strike', 'stake'],
+    [TRADE_TYPES.RISE_FALL]: {
+        duration: true,
```

Comment:
Looks a bit odd to have an object with 'true', but if you think it's better, then okay